src/main/java/Thea.java

```diff
@@ -0,0 +1,181 @@
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.File;
+
+public class Thea {
+
+    public static void main(String[] args) throws Exception {

```

Comment:
Maybe you can consider abstracting out parsing into a separate class to keep your main function shorter?